registry/server/tag_cleanup.go

```diff
@@ -0,0 +1,82 @@
+package server
+
+import (
+	"context"
+	regexp2 "regexp"
+	"strconv"
+	"time"
+)
+
+var tagMarkTimeKey = "tagMarkedForDeletionTSMin"
+var allowedTagStalenessMinutes = 60 // todo: pass this through via config
+
+// todo: should I handle errors, or just log?
+// todo: what calls this function? A timer? Should anything happen if it fails?
+// MarkForDeletion marks stored tags that have been stranded without an associated kafka resource.
+func(s *Server) MarkForDeletion(ctx context.Context) error {
+	markTimeMinutes := string(time.Now().Minute())
+
+	// Get all brokers from ZK.
+	brokers, errs := s.ZK.GetAllBrokerMeta(false)
+	if errs != nil {
+		return ErrFetchingBrokers
+	}
+
+	// Get all topics from ZK
+	rgex, _ := regexp2.Compile(".*")
+	regexes := []*regexp2.Regexp{rgex}
```

Comment:
If you don't care/don't need to re-use the slice of regexp, you can declare it anonymously in the function with: `topics, err := s.ZK.GetTopics([]*regexp2.Regexp{rgex})`